### ISSUE TITLE
wanna xfer this repo over to me or "Ether"?

### ISSUE THREAD

**Issue by JohnMcLear** [ether/ep_cursortrace#10]
Seems to be a more suitable home now as I'm maintaining it and have completely rewritten it? :)

**Comment by JohnMcLear** [comment]
Bump @redhog

**Comment by JohnMcLear** [comment]
Bump @redhog

**Comment by redhog** [comment]
Sure

**Comment by redhog** [comment]
Done :)

**Comment by JohnMcLear** [comment]
Awesome ta, can you make sure I'm able to push to the npm package too please? :)

**Comment by aultuser** [comment]
@redhog, just checking if the NPM push permissions are available for JohnMcLear.  (No rush, just a gentle reminder!)

**Comment by JohnMcLear** [comment]
```
jose@server:~/ep_cursortrace$ npm publish
npm notice
npm notice 📦  ep_cursortrace@2.0.15
npm notice === Tarball Contents ===
npm notice 51B    static/css/ace_inner.css
npm notice 1.0kB  static/css/cursortrace.css
npm notice 330B   static/js/css.js
npm notice 2.4kB  handleMessage.js
npm notice 12.5kB static/js/main.js
npm notice 589B   ep.json
npm notice 425B   package.json
npm notice 998B   README.md
npm notice 265B   .travis.yml
npm notice === Tarball Details ===
npm notice name:          ep_cursortrace
npm notice version:       2.0.15
npm notice package size:  7.1 kB
npm notice unpacked size: 18.5 kB
npm notice shasum:        19cf98998243331e32627878d566837c98d11ed5
npm notice integrity:     sha512-jA6aORk0KkuOt[...]B9N/hlrT/1TIA==
npm notice total files:   9
npm notice
npm ERR! code E403
npm ERR! 403 403 Forbidden - PUT https://registry.npmjs.org/ep_cursortrace - You cannot publish over the previously published versions: 2.0.15.
npm ERR! 403 In most cases, you or one of your dependencies are requesting
npm ERR! 403 a package version that is forbidden by your security policy.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/jose/.npm/_logs/2020-05-05T05_56_16_413Z-debug.log
```
